Dockerfile.windows

```diff
@@ -0,0 +1,49 @@
+ARG WINDOWS_VERSION=1809
+
+# Build the manager binary
+FROM golang:1.14 as builder
```

Comment:
Seems like this is running in "windows" mode. Should probably be more explicit:

```
FROM --platform=windows/amd64 golang:1.14 as builder
```

BUT it would be really nice to utilize the regular linux build, I think windows supports this now with [Linux Containers on Windows (LCOW)](https://devblogs.microsoft.com/premier-developer/running-docker-windows-and-linux-containers-simultaneously/).  I think this would allow you to move the nanoserver part to the main dockerfile for linux and then on a windows build just target the nanoserver rather than the scratch build.